manifests/c/compuphase/pawn/4.1.7152/compuphase.pawn.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: compuphase.pawn
+PackageVersion: 4.1.7152
+PackageLocale: en-US
+Publisher: Thiadmer Riemersma
```

```suggestion
Publisher: CompuPhase
```
This should match control panel